src/_content/posts/conference-wrap-up.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "2024 Conference Wrap Up"
+layout: post
+
+author: DjangoCon US Communication Team
+category: General
+published_datetime: 2024-10-07 06:00:00
+
+cover:
+    url: /assets/img/blog/Django_group_2024.jpg
+    alt: "The DjangoCon US 2024 attendees welcoming everyone to Durham, NC! It's hundreds of people standing with their arms extended outwards."
+---
+
+What a fantastic year! DjangoCon US 2024 is officially over, but has left a lasting impression. A heart-felt thank you to each of the 386 folks who attended this year. We had attendees, speakers, and sponsors from all over the world, and we are so thrilled to have such amazing representation from all the people that make up our community.
+
+DjangoCon US is a volunteer-run conference that succeeds because of the dedicated people who give up their lunches, nights, and weekends to tackle tasks small and large. We are very lucky to have so many organizers that see the value this conference brings to the tech world and are willing to donate their time to bring it to fruition. 
+
+Thank you to the amazing teams we had running our media this year! The folks from American AV, White Coat Captioning and ConFreaks were wonderful to work with. They all provided great suggestions on how we could make our video content better.
```

Comment:
```suggestion
Thank you to the amazing teams we had running our media this year! The folks from American AV, White Coat Captioning, and ConFreaks were wonderful to work with. They all provided great suggestions on how we could make our video content better.
```

Reply:
I think it would be nice lead with "ConFreaks, White Coat Captioning, and ..." 

Why? The order change is based on how long we have worked with these vendors. American AV is relatively new and Confreaks we have worked with for over a decade (they are the only vendor that predates DEFNA being DEFNA).